GDevelop.js/__tests__/GDJSAdvancedExtensionCodeGenerationIntegrationTests.js

```diff
@@ -109,4 +109,81 @@ describe('libGD.js - GDJS Code Generation integration tests', function () {
 
     expect(runtimeScene.getVariables().has('SuccessVariable')).toBe(false);
   });
+
+  it('can copy a variable parameter variable', function () {
+    const serializerElement = gd.Serializer.fromJSObject([
+      {
+        type: 'BuiltinCommonInstructions::Standard',
+        conditions: [],
+        actions: [
+          {
+            type: { value: 'CopyArgumentToVariable' },
+            parameters: ['"ParameterName"', '__MyExtensionVariable'],
+          },
+        ],
+        events: [],
+      },
+    ]);
+
+    const runCompiledEvents = generateCompiledEventsFromSerializedEvents(
+      gd,
+      serializerElement,
+      { parameterTypes: { ParameterName: 'scenevar' }, logCode: false }
+    );
+
+    const { gdjs, runtimeScene } = makeMinimalGDJSMock();
+    const myVariable = runtimeScene.getVariables().get('MyVariable');
+    myVariable.getChild('MyChildVariable').setNumber(123);
+    runCompiledEvents(gdjs, runtimeScene, [myVariable]);
+
+    // The user variable is copied into a variable with the extension namespace.
+    expect(runtimeScene.getVariables().has('__MyExtensionVariable')).toBe(true);
+    expect(
+      runtimeScene
+        .getVariables()
+        .get('__MyExtensionVariable')
+        .getChild('MyChildVariable')
+        .getAsNumber()
+    ).toBe(123);
+  });
+
+  it('can write a variable parameter variable', function () {
+    const serializerElement = gd.Serializer.fromJSObject([
+      {
+        type: 'BuiltinCommonInstructions::Standard',
+        conditions: [],
+        actions: [
+          {
+            type: { value: 'ModVarScene' },
+            parameters: ['__MyExtensionVariable.MyChildVariable', '=', '123'],
+          },
+          {
+            type: { value: 'CopyVariableToArgument' },
+            parameters: ['"ParameterName"', '__MyExtensionVariable'],
```

Comment:
Ubernit: "ParameterName" is a bit meta for the name of a parameter, it could be instead called "FirstParameter" or "SomeParameter".

```suggestion
            parameters: ['"FirstParameter"', '__MyExtensionVariable'],
```